src/services/CertificateGenerationService.ts

```diff
@@ -1381,6 +1382,24 @@ class CertificateGenerationService {
       )
     );
   }
+
+  /**
+   * Sorts required standards if present firstly by sectionNumber and then by rsNumber and then returns it
+   * @param requiredStandards - the requiredStandards array to sort
+   * @returns - the sorted requiredStandards array
+   */
+  private sortRequiredStandards = (requiredStandards: IRequiredStandard[] | undefined): IRequiredStandard[] | undefined => {
+    if (!requiredStandards) {
+      return;
+    }
+
+    const collator = new Intl.Collator("en", { numeric: true, sensitivity: "base" });
+    return requiredStandards
+        .sort((a, b) =>
+          collator.compare(a.sectionNumber, b.sectionNumber) ||
+          collator.compare(a.rsNumber.toString(), b.rsNumber.toString())
```

Comment:
Can we simplify this just to compare on ref calculation, since that is a string of `"<section>.<rsNumber><rsLetter>"` ? pedantic change admittedly.

Reply:
Oh, very interesting. Good spot.